components/brave_wallet_ui/page/screens/send/constants/magics.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+export const MAX_ZCASH_MEMO_LENGTH = 500
```

Comment:
Why is it 500, not 512?

Reply:
Sorry about that, thought it was `500`. It is now `512`!